airbrakes/data_handling/logger.py

```diff
@@ -123,55 +126,62 @@ def _convert_unknown_type(obj_type: Any) -> str:
 
     @staticmethod
     def _prepare_log_dict(
```

Comment:
this function needs to convert the state name to state letter in the context data packet